examples/RoboCasa/README.md

```diff
@@ -0,0 +1,97 @@
+| gr1_unified/PosttrainPnPNovelFromPlacematToBowlSplitA_GR1ArmsAndWaistFourierHands_Env | 0.40 |
+| gr1_unified/PosttrainPnPNovelFromPlacematToPlateSplitA_GR1ArmsAndWaistFourierHands_Env | 0.44 |
+| gr1_unified/PosttrainPnPNovelFromPlacematToTieredshelfSplitA_GR1ArmsAndWaistFourierHands_Env | 0.14 |
+| gr1_unified/PosttrainPnPNovelFromPlateToBowlSplitA_GR1ArmsAndWaistFourierHands_Env | 0.38 |
+| gr1_unified/PosttrainPnPNovelFromPlateToCardboardboxSplitA_GR1ArmsAndWaistFourierHands_Env | 0.36 |
+| gr1_unified/PosttrainPnPNovelFromPlateToPanSplitA_GR1ArmsAndWaistFourierHands_Env | 0.48 |
+| gr1_unified/PosttrainPnPNovelFromPlateToPlateSplitA_GR1ArmsAndWaistFourierHands_Env | 0.58 |
+| gr1_unified/PosttrainPnPNovelFromTrayToCardboardboxSplitA_GR1ArmsAndWaistFourierHands_Env | 0.40 |
+| gr1_unified/PosttrainPnPNovelFromTrayToPlateSplitA_GR1ArmsAndWaistFourierHands_Env | 0.44 |
+| gr1_unified/PosttrainPnPNovelFromTrayToPotSplitA_GR1ArmsAndWaistFourierHands_Env | 0.48 |
+| gr1_unified/PosttrainPnPNovelFromTrayToTieredbasketSplitA_GR1ArmsAndWaistFourierHands_Env | 0.40 |
+| gr1_unified/PosttrainPnPNovelFromTrayToTieredshelfSplitA_GR1ArmsAndWaistFourierHands_Env | 0.24 |
+| **Average**                              | **0.39**            |
+----
+
+All the above tasks are evaluated at 50 rollouts each.
+
+To evaluate, first start the inference server with our provided checkpoint:
+
+<!-- TODO: Replace with Youliang's repo. -->
+```bash
+python3 scripts/inference_service.py --server \
+    --model_path /mnt/amlfs-02/shared/checkpoints/xiaoweij/0905/robocasa-checkpoints-60K/ \
+    --data_config fourier_gr1_arms_waist
+```
+
+### Installation
+
+Follow the [official RoboCasa installation guide](https://github.com/robocasa/robocasa-gr1-tabletop-tasks?tab=readme-ov-file#getting-started).
+
+Then run the evaluation:
+```bash
+python3 scripts/simulation_service.py --client \
+    --env_name <TASK_NAME> \
+    --video_dir ./videos \
+    --max_episode_steps 720 \
+    --n_episodes 50
+```
+
+----
+
+## Reproduce Training Results
+
+To reproduce the training results, you can use the following steps:
+1. Download the datasets
+2. Fine-tune the model
+3. Evaluate the model (same as above)
+
+### 📦 1. Dataset Preparation
+
+<!-- TODO: Upload the 1K per traj data to HF. And add instructions -->
+
+Download LeRobot-compatible datasets directly from Hugging Face.
+
+Modality config is already pre-configured for you in the dataset.
+
+### 🚀 Model Fine-tuning
+
+The fine-tuning script supports multiple configurations.
+
+```bash
+python scripts/gr00t_finetune.py \
+    --dataset-path /mnt/amlfs-03/shared/datasets/lerobot/78dc_fixed_annotations/ \
+    --num-gpus 8 \
+    --batch-size 64 \
+    --learning_rate 3e-5 \
+    --output-dir /tmp/robocasa-checkpoint \
+    --data-config fourier_gr1_arms_waist \
+    --embodiment_tag gr1 \
+    --tune-visual \
+    --max-steps 30000 \
+    --save-steps 5000
+```
```

Comment:
try with:
```
python scripts/gr00t_finetune.py \
  --dataset-path "${ALL_DATASET_PATHS[@]}" \
  --num-gpus 8 --batch-size 60 --learning_rate 3e-5 \
  --output-dir /mnt/amlfs-03/shared/youliangt/checkpoints/24dc-all-60k-2aug-tunevisual \
  --data-config fourier_gr1_arms_waist --embodiment_tag gr1 --tune-visual \
  --max-steps 60000 --save-steps 5000
```

Reply:
Yes, this is outdated. I tried with the same hyperparameter like you mentioned here.